src/lib.rs

```diff
@@ -122,9 +122,7 @@ impl Bme280 {
             sda_pullup_en: true,
             scl_io_num: scl_pin,
             scl_pullup_en: true,
-            __bindgen_anon_1: i2c_config_t__bindgen_ty_1 {
-                master: i2c_config_t__bindgen_ty_1__bindgen_ty_1 { clk_speed: 100000 },
-            },
+            master: i2c_config_t__bindgen_ty_1  { clk_speed: 100000 },
```

Comment:
Matt I'd like to look at this change and the one below with you, because I'm actually a little unclear on what was going on here in the first place, and I don't have high confidence in this fix.

Reply:
@mattjperez - We discussed in person. Are you good with moving forward with this as is?

Reply:
yup, all good. 

notes from discussion: the `i2c_config_t` here is a wrapper from [`espressif/esp-iot-solution`](https://github.com/espressif/esp-iot-solution/tree/72358cc1a1c9825a9859e1b509176644634f263b/components/i2c_bus) that underneath uses the ESP-IDF i2c interfaces